test/ygrunner.py

```diff
@@ -173,6 +173,27 @@ def test_start_ygs(setup_ygrunner, num_ygs, wallet_structures, fb_indices,
             ygclass = DeterministicMaliciousYieldGenerator
         else:
             ygclass = MaliciousYieldGenerator
+    # Note for tests of Lightning message channels:
```

Comment:
Note to self: change and/or remove this comment section.

Reply:
Removed  in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1182/commits/aacc4f53a9efabf7653b36d4fe613b4e8e06da7b